guides/common/modules/proc_installing-postgresql.adoc

```diff
@@ -24,13 +23,13 @@ rh-postgresql12-postgresql-evr
 # postgresql-setup initdb
 ----
 +
-. Edit the `/var/opt/rh/rh-postgresql12/lib/pgsql/data/postgresql.conf` file:
+. Edit the `/var/lib/pgsql/data/postgresql.conf` file:
 +
 [options="nowrap" subs="verbatim,quotes"]
 ----
-# vi /var/opt/rh/rh-postgresql12/lib/pgsql/data/postgresql.conf
+# vi /var/lib/pgsql/data/postgresql.conf
 ----
-+
+
```

Comment:
It said it can't generate the preview. Perhaps this `+` should not have been removed?